src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/processors/PrisonerUpdatedEventProcessor.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.personrecord.message.listeners.processors
+
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.personrecord.model.DomainEvent
+
+const val PRISONER_UPDATED = "probation-case.engagement.created"
```

Comment:
if we do end up sticking with 2 processors, this value needs changing

Reply:
Yes, copy/past error.